README.md

```diff
@@ -1,36 +1,45 @@
 - Testing on a 1225v6 Xeon (7th Gen Kaby Lake) used ~2-3% CPU for a 1080p VC-1 -> 1080p H.264 hardware transcode. The same machine was at 70-75% CPU for a software transcode of VC-1 -> H.264. Some Bluray disks used VC-1, before H.264 became ubiquitous.
 - Testing on a 1225v6 Xeon (7th Gen Kaby Lake) used ~4% CPU for a 4k HDR HEVC 10-bit -> 1080p SDR H.264 hardware transcode. The same machine was at 100% CPU for a software transcode of the same file. Running four 4k hardware transcodes at once did not significantly tax the machine, further testing to determine the max number of possible 4k transcodes was not done.
 - Testing on a G4560 Pentium (7th Gen Kaby Lake) used ~30% CPU for a 4k HEVC -> 1080p H.264 transcode.
 - Testing on a Core i3 6100T (6th Gen SkyLake) shows that acceleration of a 4k HEVC -> 1080p H.264 transcode is minimal on it and takes 95% CPU. This is likely due to limited HEVC features in that generation.
-- Core 4th Gen (Haswell) added H.265 and VC1 support in QuickSync. For 1080p transcode, "Haswell or better" is reasonable guidance.
+- Core 4th Gen (Haswell) added H.264 and VC1 support in QuickSync. For 1080p transcode, "Haswell or better" is reasonable guidance.
 - The Kaby Lake iGPU improved HEVC acceleration and added 10-bit (HDR) HEVC. The HEVC (H.265) codec is used on UHD Blurays. Consider a Kaby Lake or later CPU a requirement for significant acceleration of HEVC (Bluray 4k UHD) content. Note that without HDR10 tone mapping, 4k HDR transcode remains suboptimal, and it is often better to prepare a good SDR 1080p file ahead of time. HDR10 tone mapping is not yet available in a Desktop / Server CPU/iGPU combination.
 ## Compatibility
-- The current (11.3-Ux) Freenas Drivers support Intel CPU Generation 2-7, Sandy Bridge through Kaby Lake
-- The current (11.3-Ux) Freenas Drivers DO NOT support Intel CPU Generation 8 and 9. This means Coffee Lake and later, Intel Xeon E-xxxxG and Intel Core i3/i5/i7 8xxx and later.
-- Support for Generation 8 and 9 can not be added without a kernel upgrade to 12.x. Don't bother trying to compile it from source. TrueNAS 12.x Core will gain driver support for the iGPUs in 8th and 9th gen CPUs.
+- The 11.3-Ux FreeNAS drivers support the iGPU in Intel CPU Generation 2-7, Sandy Bridge through Kaby Lake
+- The 11.3-Ux FreeNAS drivers DO NOT support the iGPU in Intel CPU Generation 8 and 9. This means Coffee Lake and later, Intel Xeon E-xxxxG and Intel Core i3/i5/i7/i9 8xxx and later.
+- Support for the iGPU in generation 8 and 9 CPUs can not be added without a kernel upgrade to 12.x. Preliminary testing shows that TrueNAS Core 12.0 / FreeBSD 12.1 supports the iGPUs in 8th generation CPUs (Coffee Lake i3/5/7/9-8xxx and related Pentium, Xeon E-21xxG), does not support the iGPUs in 9th generation desktop CPUs (Coffee Lake Refresh i3/5/7/9-9xxx and related Pentium), and likely does support the iGPUs in 9th generation entry-level Xeon CPUs (Xeon E-22xxG).
```

Comment:
Supprt for 9th gen should be in the 12 kernel, 8 and 9th gen drivers support was the same commit...

Reply:
These are the initial support pushes:
https://github.com/FreeBSDDesktop/kms-drm/commit/1f2bfbcd97c71f919f492dea488b7a4efb8a432e
https://github.com/FreeBSDDesktop/kms-drm/commit/fc623cac8b9bd0e0440fa3a42fe8b0fa16335019

Reply:
For FreeNAS 12 the drm-kmod is latest: g20200221